### PR TITLE
Update test-install-oracle.yaml

### PR DESCRIPTION
**Pull request by jcnars**
Adding official ansible image and also moving the presubmit command into a file in a separate location, avoiding the need to define in the job itself the various arguments that's typically needed for the `install-oracle.sh`

### CODE REVIEW

File: prow/prowjobs/google/bms-toolkit/test-install-oracle.yaml [hunk 1]
```diff
@@ -6,14 +6,11 @@ presubmits:
     decorate: true
     spec:
       containers:
-      - image: gcr.io/google.com/cloudsdktool/cloud-sdk:alpine
+      - image: quay.io/ansible/ansible-runner:latest
```
Comment (mfielding) [inline prow/prowjobs/google/bms-toolkit/test-install-oracle.yaml:9]: From JC offline, he mentions that this is actually an official RedHat repo

Comment (mfielding) [inline prow/prowjobs/google/bms-toolkit/test-install-oracle.yaml:9]: Given that, according to the README, we support Ansible 2.9 or higher, can we tests against this ansible version?  That way we can help ensure we keep this support.

File: prow/prowjobs/google/bms-toolkit/test-install-oracle.yaml [hunk 2]
```diff
@@ -6,14 +6,11 @@ presubmits:
...
         command:
-        - /bin/bash
-        args:
-        # probing the pod created
-        - "true && id && echo && hostname && echo && pwd && echo && ls -l && echo && whoami && echo && env && echo && df -h"          
+        - ./presubmit_tests/test_pr.sh
```
Comment (mfielding) [inline prow/prowjobs/google/bms-toolkit/test-install-oracle.yaml:11]: Any way to qualify this directory?  (Also, where does it come from?  I don't think see a test_pr.sh in this directory)

File: prow/prowjobs/google/bms-toolkit/test-install-oracle.yaml (hunk 1 above)
Comment (jcnars) [reply prow/prowjobs/google/bms-toolkit/test-install-oracle.yaml:9]: Links:
* [quay.io ](https://quay.io/) is Redhat's own container registry (https://access.redhat.com/articles/5925591).
* The Ansible org inside it is: https://quay.io/organization/ansible that provides the updates to the aforementioned  image.
* Ansible's official [dockerhub page](https://hub.docker.com/r/ansible/ansible-runner) reads: 
```
This repository is obsolete. The official Ansible Runner images now live at quay.io/ansible/ansible-runner
```

Comment (jcnars) [reply prow/prowjobs/google/bms-toolkit/test-install-oracle.yaml:9]: Modifying it to use 2.9 stable version.
Yes, the testing did work for me successfully from inside a GKE pod:
```
jcnarasimhan@jon2:~/DriveFS/My Drive/bmaas/test_automation/oss_prow_onboarding/running_cleanup_install_oracle$ kubectl exec -it test-pods-namespace-lo-podname -n test-pods -- /bin/bash
bash-4.4# cd /root/bms-toolkit/
bash-4.4# ls -l sydney-1.out.thegabba.brutecleanup 
-rw-r--r-- 1 root root 24974 Nov 16 02:03 sydney-1.out.thegabba.brutecleanup
bash-4.4# tail sydney-1.out.thegabba.brutecleanup 

TASK [brute-ora-cleanup : Server reboot] ***************************************
skipping: [linuxserver44.orcl]

TASK [brute-ora-cleanup : Re-mount swlib] **************************************
skipping: [linuxserver44.orcl]

PLAY RECAP *********************************************************************
linuxserver44.orcl         : ok=21   changed=8    unreachable=0    failed=0    skipped=14   rescued=0    ignored=5   
```

File: prow/prowjobs/google/bms-toolkit/test-install-oracle.yaml (hunk 2 above)
Comment (jcnars) [reply prow/prowjobs/google/bms-toolkit/test-install-oracle.yaml:11]: `    decorate: true` directive will clone our bms-toolkit repository **with** the proposed PR and cd into it.
Hence the `.` represents the base directory of the bms-toolkit repo, which is `bms-toolkit`, like:
```
jcnarasimhan@jon2:~/DriveFS/My Drive/bmaas/test_automation/oss_prow_onboarding/running_cleanup_install_oracle$ kubectl exec -it test-pods-namespace-lo-podname -n test-pods -- /bin/bash
bash-4.4# cd /root
bash-4.4# ls -l
total 16
-rw------- 1 root root 2361 Sep 15 14:17 anaconda-ks.cfg
-rw-r--r-- 1 root root  608 Sep 15 14:17 anaconda-post.log
drwxr-xr-x 7 root root 4096 Nov 16 02:13 bms-toolkit
-rw------- 1 root root 2059 Sep 15 14:17 original-ks.cfg
bash-4.4# cd bms-toolkit/
bash-4.4# ls
LICENSE      apply-patch.sh	   check-swlib.sh	config-db.yml		 group_vars	     install-sw.yml	      roles
OWNERS	     asm_disk_config.json  check-swlib.yml	config-rac-db.yml	 host-provision.sh   manage_devices_on_vm.sh  sydney-1.out.thegabba.brutecleanup
README.md    brute-cleanup.yml	   cleanup-oracle.sh	data_mounts_config.json  host-provision.yml  patch.yml
ansible.cfg  check-instance.yml    cluster_config.json	docs			 install-oracle.sh   prep-host.yml
```

I will, in my testing PR, create the ${bms-toolkit-root-dir}/presubmit_tests/test_pr.sh

**Comment by jcnars** [comment]
> Adding official ansible image and also moving the presubmit command into a file in a separate location, avoiding the need to define in the job itself the various arguments that's typically needed for the `install-oracle.sh`

Thanks for the review - I have provided the updates and it's ready for your further review/lgtm.

File: prow/prowjobs/google/bms-toolkit/test-install-oracle.yaml [hunk 3]
```diff
@@ -6,14 +6,11 @@ presubmits:
     decorate: true
     spec:
       containers:
-      - image: gcr.io/google.com/cloudsdktool/cloud-sdk:alpine
+      - image: quay.io/ansible/ansible-runner:stable-2.9-latest
```
Comment (jcnars) [inline prow/prowjobs/google/bms-toolkit/test-install-oracle.yaml:9]: `:stable-2.9-latest` tag was successfully tested.
Using `:latest` tag throws following warning:
```
[WARNING]: You are running the development version of Ansible. You should only
run Ansible from "devel" if you are modifying the Ansible engine, or trying out
features under development. This is a rapidly changing source of code and can
become unstable at any point.
```

File: prow/prowjobs/google/bms-toolkit/test-install-oracle.yaml (hunk 2 above)
Comment (mfielding) [reply prow/prowjobs/google/bms-toolkit/test-install-oracle.yaml:11]: Is it possible to note this in the code itself?  It doesn't seem obvious.  Perhaps a github link to the script itself.  And is it possible to qualify the path to be executed from?  perhaps there might be an environment that already sets this.

Comment (jcnars) [reply prow/prowjobs/google/bms-toolkit/test-install-oracle.yaml:11]: Sure, adding note to the code.
The script in bms-toolkit repo is non-existent at this point, but will be created when testing this prowjob.
I am not aware of environment variable that prow provides to specify the root directory.

Comment (jcnars) [reply prow/prowjobs/google/bms-toolkit/test-install-oracle.yaml:14]: Created the <directory>/<.sh> script in a separate PR: https://github.com/google/bms-toolkit/pull/95

File: prow/prowjobs/google/bms-toolkit/test-install-oracle.yaml [hunk 4]
```diff
@@ -5,15 +5,48 @@ presubmits:
...
+        # the presubmit_tests dir and test_pr.sh are non-existent at this point, 
+        # but will be created in a test PR as part of testing the prowjob
+        - ./presubmit_tests/test_pr.sh        
         resources:
           requests:
             memory: "2.0Gi"
             cpu: "3.0"
-       
\ No newline at end of file
+        volumeMounts:
```
Comment (jcnars) [inline prow/prowjobs/google/bms-toolkit/test-install-oracle.yaml:20]: These volumes/volumeMounts directives are kubectl-tested as in: (internal): http://b/202240337#comment22

**Comment by fejta** [comment]
/uncc
(feel free to self manage this PR, add me back if my help is needed)

**Comment by jcnars** [comment]
/cc fejta
Hello, could you please eye-over the changed file from a kubernetes / prowjobs syntax perspective?

You don't have to attach the formal /lgtm label, just a lgtm from prow presubmit job point of view?

Thanks

**Comment by jcnars** [comment]
The corresponding k8s manifest I used to test this locally is at:  https://gist.github.com/jcnars/88994d0306fcd796efd7df0cc883514a

**Review by mfielding — APPROVED** [review]

**Pull request merged**